common/src/main/java/com/revenuecat/purchases/common/OfferingFactories.kt

```diff
@@ -11,7 +11,7 @@ import org.json.JSONObject
 /**
  * Note: this may return an empty Offerings.
  */
-fun JSONObject.createOfferings(productsById: Map<String, StoreProduct>): Offerings {
+fun JSONObject.createOfferings(productsById: Map<String, List<StoreProduct>>): Offerings {
```

Comment:
I think we should be seeing a single StoreProduct now, but with multiple offers, right?